### PR TITLE
Updated badges for v3

### PR DESCRIPTION
**Pull request by meatballhat**
## What type of PR is this?

- cleanup
- documentation

## What this PR does / why we need it:

Updates badges to correctly target main branch / v3 module.

## Which issue(s) this PR fixes:

NA 😬

### CODE REVIEW

**Comment by abitrolly** [comment]
What is not right with Codebeat? Seem to complain a lot about complexity https://codebeat.co/projects/github-com-urfave-cli

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # cli
 
-[![GoDoc](https://godoc.org/github.com/urfave/cli?status.svg)](https://pkg.go.dev/github.com/urfave/cli/v3)
-[![codebeat](https://codebeat.co/badges/0a8f30aa-f975-404b-b878-5fab3ae1cc5f)](https://codebeat.co/projects/github-com-urfave-cli)
-[![Go Report Card](https://goreportcard.com/badge/urfave/cli)](https://goreportcard.com/report/urfave/cli)
-[![codecov](https://codecov.io/gh/urfave/cli/branch/main/graph/badge.svg)](https://codecov.io/gh/urfave/cli)
+[![Run Tests](https://github.com/urfave/cli/actions/workflows/cli.yml/badge.svg)](https://github.com/urfave/cli/actions/workflows/cli.yml)
+[![Go Reference](https://pkg.go.dev/badge/github.com/urfave/cli/v3.svg)](https://pkg.go.dev/github.com/urfave/cli/v3)
+[![Go Report Card](https://goreportcard.com/badge/github.com/urfave/cli/v3)](https://goreportcard.com/report/github.com/urfave/cli/v3)
+[![codecov](https://codecov.io/gh/urfave/cli/branch/main/graph/badge.svg)](https://app.codecov.io/gh/urfave/cli)
```
Comment (skelouse) [inline README.md:6]: This link is not pointing to the main branch: https://app.codecov.io/gh/urfave/cli

Comment (meatballhat) [reply README.md:6]: Yep I'm not sure what's going on there yet 🙃 Looking soon!

Comment (meatballhat) [reply README.md]: Welp, I did nothing and now it looks better (?)

**Review by skelouse — APPROVED** [review]

**Pull request merged**